docs/docs/authentication.md

```diff
@@ -4,98 +4,120 @@ description: Set up an authentication provider
 
 # Authentication
 
-`@redwoodjs/auth` contains both a built-in database-backed authentication system (dbAuth), as well as lightweight wrappers around popular SPA authentication libraries.
+Redwood has integrated auth end to end, from the web side to the api side.
+On the web side, the router can protect pages via the `Private` component, and even restrict access at the role-level.
```

Comment:
Could you also mention the `private` prop on `<Set>` here? Mainly to save people from doing
```
<Private unauth...
  <Set wra...
```
or
```
<Set wra...
  <Private una...
```

Reply:
Sure, see https://github.com/redwoodjs/redwood/pull/7229/commits/ee1fbdced38696a63a2a933402631a28eacac78d